UNIT64_MAX

你好，nihui，当我在centos7.2编译最新版本的ncnn时，报出了下面的错误：
src/command.cpp:2217:95: error: ‘UINT64_MAX’ was not declared in this scope
之前旧的ncnn版本编译都没有问题，请问如何解决呢？谢谢，

--- a/src/command.cpp
+++ b/src/command.cpp
@@ -2214,7 +2214,7 @@ int VkCompute::submit_and_wait()
 
     // wait
     {
-        VkResult ret = vkWaitForFences(vkdev->vkdevice(), 1, &compute_command_fence, VK_TRUE, UINT64_MAX);
+        VkResult ret = vkWaitForFences(vkdev->vkdevice(), 1, &compute_command_fence, VK_TRUE, (uint64_t)-1);
         if (ret != VK_SUCCESS)
         {
             NCNN_LOGE("vkWaitForFences failed %d", ret);
@@ -2990,7 +2990,7 @@ int VkTransfer::submit_and_wait()
     // wait
     if (vkdev->info.unified_compute_transfer_queue)
     {
-        VkResult ret = vkWaitForFences(vkdev->vkdevice(), 1, &compute_command_fence, VK_TRUE, UINT64_MAX);
+        VkResult ret = vkWaitForFences(vkdev->vkdevice(), 1, &compute_command_fence, VK_TRUE, (uint64_t)-1);
         if (ret != VK_SUCCESS)
         {
             NCNN_LOGE("vkWaitForFences failed %d", ret);
@@ -3001,7 +3001,7 @@ int VkTransfer::submit_and_wait()
     {
         VkFence fences[2] = {upload_command_fence, compute_command_fence};
 
-        VkResult ret = vkWaitForFences(vkdev->vkdevice(), 2, fences, VK_TRUE, UINT64_MAX);
+        VkResult ret = vkWaitForFences(vkdev->vkdevice(), 2, fences, VK_TRUE, (uint64_t)-1);
         if (ret != VK_SUCCESS)
         {
             NCNN_LOGE("vkWaitForFences failed %d", ret);

